packages/sqflite/lib/src/sentry_sqflite_database_factory.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:meta/meta.dart';
 import 'package:sentry/sentry.dart';
 import 'package:sqflite/sqflite.dart';
+import 'package:sqflite/sqflite.dart' as sqflite show databaseFactory;
```

Comment:
Import is now "duplicated"

```Dart
import 'package:sqflite/sqflite.dart';
import 'package:sqflite/sqflite.dart' as sqflite show databaseFactory;
```

Could we just use the latter with the `sqflite` prefix?

Reply:
I can change it so all sqflite calls in this file use the sqflite prefix, yea